examples/ephemeral-spark/rds.tf

```diff
@@ -5,7 +5,7 @@ resource "random_password" "rds-password" {
 }
 
 module "rds-postgres" {
-  source = "git::git@github.com:Datatamer/terraform-aws-rds-postgres.git?ref=1.0.0"
+  source = "git::git@github.com:Datatamer/terraform-aws-rds-postgres.git?ref=2.1.0"
```

Comment:
The 3.0.0 release has an important update to the default value for `engine_version`, due to RDS auto upgrading patches.
https://github.com/Datatamer/terraform-aws-rds-postgres/releases/tag/3.0.0

So you can either upgrade to 3.0.0 or just set `engine_version` to "12" instead of the old default of "12.3" since the latest version is "12.5".

Ran into an issue when RDS upgrades patch versions, it causes the terraform state to get out of sync.

Reply:
oh ok! I'll go with 3.0.0.